### PR TITLE
CI: Checkstyle

### PR DESCRIPTION
**Pull request by ericenns**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Added in checkstyle config and fixed issues discovered by it.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [ ] Tests added (or description of how to test) for any new features.
* [ ] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/java/ca/corefacility/bioinformatics/irida/oauth2/OAuth2ResourceOwnerPasswordAuthenticationProvider.java [hunk 1]
```diff
@@ -37,55 +37,55 @@
...
-        // @formatter:off
+	private static final String ERROR_URI = "https://datatracker.ietf.org/doc/html/rfc6749#section-5.2";
+	private final AuthenticationManager authenticationManager;
+	private final OAuth2AuthorizationService authorizationService;
+	private final OAuth2TokenGenerator<? extends OAuth2Token> tokenGenerator;
+
+	/**
+	 * Constructs an {@code OAuth2ResourceOwnerPasswordAuthenticationProvider} using the provided parameters.
+	 */
+	public OAuth2ResourceOwnerPasswordAuthenticationProvider(AuthenticationManager authenticationManager,
```
[the remark this answers is not in the document]
Comment (ericenns) [reply src/main/java/ca/corefacility/bioinformatics/irida/oauth2/OAuth2ResourceOwnerPasswordAuthenticationProvider.java:52]: Fixed in 0c3d988c885dc3cda57a5f3182602a5cd50ccd41

**Review by joshsadam — APPROVED** [review]
I actually do not have much to say about this except that I really appreciate you doing it.  I like that you separated out linting into its own file.  I am assuming, from what I looked through, that **all** the changes to the other `.java` files were to fix the linting issues.  This is also greatly appreciated and should really help simplify PRs.

**Review by deepsidhu85 — COMMENTED** [review]
Looks good..thanks for adding this! Just want to confirm that we will be updating it after to only run on files that were changed. Unless I missed it

**Comment by ericenns** [comment]
> Looks good..thanks for adding this! Just want to confirm that we will be updating it after to only run on files that were changed. Unless I missed it

Yeah by default it only comments and fails based on the diff of the pr. I.e. only on files actually updated by the pr.

**Comment by deepsidhu85** [comment]
> comments and fails based on the diff o

Perfect..thanks! @ericenns

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**